builder/src/main/java/cz/xtf/builder/db/MsSQL.java

```diff
@@ -0,0 +1,120 @@
+package cz.xtf.builder.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import cz.xtf.builder.builders.pod.ContainerBuilder;
+import cz.xtf.builder.builders.pod.PersistentVolumeClaim;
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.image.Image;
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShifts;
+import io.fabric8.kubernetes.api.builder.Visitor;
+import io.fabric8.kubernetes.api.model.ServiceAccountBuilder;
+import io.fabric8.openshift.api.model.SecurityContextConstraintsBuilder;
+
+/**
+ * Note that MsSQL Server database needs to run with custom user id on Openshift (0 should do the trick)
+ * This means that this class is only usable on clusters with admin privileges to be able to add securityContextConstraint
+ *
+ * Also, the MsSQL image doesn't provide any option to add custom database and user on setup.
+ *
+ * By using this class you are accepting the Microsoft's EULA: https://go.microsoft.com/fwlink/?linkid=857698
+ */
+public class MsSQL extends AbstractSQLDatabase {
+
+    private final String ANYUID_SERVICE_ACCOUNT = "anyuid-sa";
+    private final long USER_ID = 0;
+    private static final String USERNAME = "sa";
+    private static final String PASSWORD = "myMsSQLPassword123";
+    private static final String DB_NAME = "master";
+    private static final String SYMBOLIC_NAME = "MSSQL";
+    private static final String DATA_DIR = "/var/lib/mssql/data";
+
+    public MsSQL() {
+        super(USERNAME, PASSWORD, DB_NAME, SYMBOLIC_NAME, DATA_DIR);
+    }
+
+    public MsSQL(boolean withLivenessProbe, boolean withReadinessProbe) {
+        super(USERNAME, PASSWORD, DB_NAME, SYMBOLIC_NAME, DATA_DIR, withLivenessProbe, withReadinessProbe);
+    }
+
+    public MsSQL(PersistentVolumeClaim pvc) {
+        super(USERNAME, PASSWORD, DB_NAME, SYMBOLIC_NAME, DATA_DIR, pvc);
+    }
+
+    public MsSQL(PersistentVolumeClaim pvc, boolean withLivenessProbe, boolean withReadinessProbe) {
+        super(USERNAME, PASSWORD, DB_NAME, SYMBOLIC_NAME, DATA_DIR, pvc, withLivenessProbe, withReadinessProbe);
+    }
+
+    public MsSQL(String username, String password, String dbName) {
+        super(username, password, dbName, SYMBOLIC_NAME, DATA_DIR);
+    }
+
+    public MsSQL(String symbolicName, boolean withLivenessProbe, boolean withReadinessProbe) {
+        super(USERNAME, PASSWORD, DB_NAME, symbolicName, DATA_DIR, withLivenessProbe, withReadinessProbe);
+    }
+
+    @Override
+    protected void configureContainer(ContainerBuilder containerBuilder) {
+        super.configureContainer(containerBuilder);
+        
+        //Setting up the anyuid service account
+        //Don't close the OpenShift Client instance, it could cause trouble with pod shell access
```

Comment:
Not sure we want this info here, it is a general note more suited for README or maybe `OpenShifts.master()` javadoc .. but it isn't a socpe of this PR at all

Reply:
I added this comment just in case someone else would see this line and wanted to use the `try with resource` block as it's suggested by IDEA.
Agree that it should be mentioned elsewhere though.

Reply:
If IDEA is suggesting something we know can bring problem, can it be fixed in XTF code? Can we handle it as XTF bug?

Reply:
you can skip the IDEA from suggesting this by introducing comment
```suggestion
        //noinspection resource
```

But IDEA is not the only IDE in the world so I don't know if we want to go this road